Packs/NetscoutAED/Integrations/NetscoutAED/NetscoutAED.yml

```diff
@@ -0,0 +1,1286 @@
+      type: Unknown
+    - contextPath: NetscoutAED.InboundBlacklistUrl.update_time
+      description: The time that the domain was added to the list.
+      type: Date
+  - arguments:
+    - default: false
+      description: A specific central configuration ID or -1 for global. Cannot be
+        used with the pgid parameter.
+      isArray: false
+      name: cid
+      required: false
+      secret: false
+    - default: false
+      description: A specific protection group ID or -1 for global. Cannot be used
+        with the cid parameter.
+      isArray: false
+      name: pgid
+      required: false
+      secret: false
+    - default: false
+      description: URL or a comma-separated list of URLs to add.
+      isArray: false
+      name: url
+      required: true
+      secret: false
+    - default: false
+      description: A message to associate with each URL that you add to the blacklist.
+      isArray: false
+      name: annotation
+      required: false
+      secret: false
+    deprecated: false
+    description: Adds one or more URLs to the blacklist by pgid or cid.
+    execution: false
+    name: na-aed-inbound-blacklisted-urls-add
+    outputs:
+    - contextPath: NetscoutAED.InboundBlacklistUrl.annotation
+      description: List of messages associated with each url in the inbound blacklist.
+      type: Unknown
+    - contextPath: NetscoutAED.InboundBlacklistUrl.cid
+      description: List of central configuration IDs.
+      type: Unknown
+    - contextPath: NetscoutAED.InboundBlacklistUrl.url
+      description: URL address.
+      type: String
+    - contextPath: NetscoutAED.InboundBlacklistUrl.pgid
+      description: List of protection group IDs.
+      type: Unknown
+    - contextPath: NetscoutAED.InboundBlacklistUrl.update_time
+      description: The time that the domain was added to the list.
+      type: Date
+  - arguments:
+    - default: false
+      description: URL or a comma-separated list of URLs.
+      isArray: false
+      name: url
+      required: true
+      secret: false
+    deprecated: false
+    description: Removes one or more URLs from the blacklist for a specific protection
+      group or for all protection groups.
+    execution: false
+    name: na-aed-inbound-blacklisted-urls-remove
+  - arguments:
+    - default: false
+      description: A single IPv4 host address or CIDR, or a comma-separated list of
+        IPv4 host addresses or CIDRs to remove.
+      isArray: false
+      name: host_address
+      required: true
+      secret: false
+    deprecated: false
+    description: Removes one or more hosts or CIDRs from the outbound whitelist.
+    execution: false
+    name: na-aed-outbound-whitelisted-hosts-remove
+  dockerimage: demisto/python3:3.9.5.20070
+  feed: false
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+tests:
+- No tests (auto formatted)
```

Comment:
Please replace this with test playbook you wrote.

Reply:
Done